### PR TITLE
To detect TTY in nebula console

### PR DESCRIPTION
**Pull request by dutor**
When `stdin` is not detached to a TTY, command history will not be saved by default, and no prompt will be printed.

### CODE REVIEW

File: src/console/CliManager.cpp [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 DECLARE_string(u);
 DECLARE_string(p);
+DEFINE_bool(enable_history, false, "Whether to force saving the command history");
```
Comment (steppenwolfyuetong) [inline src/console/CliManager.cpp]: Great God, I have a question.
This ".nebula_history" file is used to save history commands. Is there a file size limit? When the file is large, does it take a long time to start initialization (loadHistory)?

How did you think about this? Thx.

Comment (dutor) [reply src/console/CliManager.cpp]: This is good question. We will take good care of this, in another PR.

Comment (dutor) [reply src/console/CliManager.cpp]: This is an option to **force** the history output, even no tty attached. With this option set to false, the history saving is automatical.

Comment (monadbobo) [reply src/console/CliManager.cpp]: OK, I get it.

**Pull request merged**